devpack/src/main/java/io/neow3j/devpack/Iterator.java

```diff
@@ -15,11 +15,13 @@
  * would have to first put them into a {@link List} or a {@link Map} as shown in the following
  * example:
  * <pre>
- * {@literal ...}
- * {@literal List<ByteString> list = new io.neow3j.devpack.List<>()};
- * {@literal iterator.next()};
- * {@literal list.add(iterator.getValue());}
- * {@literal ...}
+ * {@literal
+ * ...
+ *  List<ByteString> list = new io.neow3j.devpack.List<>();
```

Comment:
Please, revert to `@literal`. 

The `@literal` is used to avoid warnings about encoding and strange chars like `<` and `>` in the JavaDoc.

If you use `./gradlew javadoc` you will see all the related warnings.

Reply:
I think this was a merge conflict at some point...
Anyway, the `{@literal}` is wrapped around the whole code example. I checked with `./graldew javadoc` and it builds without issues.

Reply:
OH, my fault! I haven't seen that the `@literal` was multi-line! Awesome. All fine. 😄